### PR TITLE
Mnist fixes

### PR DESCRIPTION
**Pull request by DanSanche**
While in the process of updating the codelab, I ran into a number of issues in the mnist example README file, which I attempted to fix here

**Some of the fixes:**
- training image path not set
- modelPath defaults to a gcs bucket in "local" section
- bucket wasn't set up
- inconsistent use of KSENV vs ENV
- EXPORT_DIR wasn't set
- port forward command wasn't working properly
- mnist-service is not deployed
- issues in the markdown rendering
- personal environments checked in to ks_app

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/examples/495)

### CODE REVIEW

**Comment by DanSanche** [comment]
@texasmichelle Do you have any time to review this?

File: mnist/README.md [hunk 1]
```diff
@@ -508,18 +527,18 @@ URI; if not you can always copy it to GCS using `gsutil`.
...
 The output should look something like
 
 ```
-gs://${EXPORT_DIR}/1547100373/saved_model.pb
-gs://${EXPORT_DIR}/1547100373/variables/:
-gs://${EXPORT_DIR}/1547100373/variables/
-gs://${EXPORT_DIR}/1547100373/variables/variables.data-00000-of-00001
-gs://${EXPORT_DIR}/1547100373/variables/variables.index
+${EXPORT_DIR}/1547100373/saved_model.pb
+${EXPORT_DIR}/variables/:
```
Comment (lluunn) [inline mnist/README.md]: Shouldn't variables dir also under 1547100373?

File: mnist/README.md [hunk 2]
```diff
@@ -528,14 +547,13 @@ The number `1547100373` is a version number auto-generated by TensorFlow; it wil
 Set your model path
 
 ```
-ks param set ${ENV} mnist-deploy-gcp modelBasePath ${EXPORT_DIR}
-
+ks param set --env=${KSENV} mnist-deploy-gcp modelBasePath ${EXPORT_DIR}
```
Comment (lluunn) [inline mnist/README.md]: --env=${KSENV} is optional

File: mnist/README.md [hunk 3]
```diff
@@ -141,10 +154,20 @@ and running distributed.
 
 ```
 KSENV=distributed
-cd ks_app
```
Comment (lluunn) [inline mnist/README.md]: still need the cd command?

File: mnist/README.md [hunk 4]
```diff
@@ -183,7 +207,7 @@ then a number of steps have already been performed for you
      * To see the secrets in your cluster
      
        ```
-       kubectl get secrets
+       kubectl get secrets --all-namespaces
```
Comment (lluunn) [inline mnist/README.md]: "-n kubeflow" should be enough?

File: mnist/README.md (hunk 3 above)
Comment (DanSanche) [reply mnist/README.md]: I was expecting that they will run through the document in order, and they should already be in the ks_app directory from the previous section. I guess it's possible some people may skip straight to this step, so the extra cd might be helpful. Either way is fine with me

File: mnist/README.md (hunk 4 above)
Comment (DanSanche) [reply mnist/README.md]: yeah good point, I'll change it

File: mnist/README.md (hunk 1 above)
Comment (DanSanche) [reply mnist/README.md]: good catch

File: mnist/README.md (hunk 2 above)
Comment (DanSanche) [reply mnist/README.md]: every other line in the tutorial used --env=${KSENV}, so I wanted to make sure this was consistent. Do you think we should mention it's optional somewhere?

Comment (lluunn) [reply mnist/README.md]: I see, then that's fine

**Comment by DanSanche** [comment]
@lluunn mind doing another lgtm? I was having the same lint issues as https://github.com/kubeflow/examples/pull/492. I tried another merge from master, so hopefully that should fix it

**Pull request merged**